.tekton/downstream-ImageContentSourcePolicy.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: acs-redhat-io
+spec:
+  repositoryDigestMirrors:
```

Comment:
What's missing here is the entry for the operator-bundle itself. When we release FBCs, we'll follow the same idea: first we build bundle to quay.io/rhacs-eng/, get its digest but add entry to FBC as if this image with this digest is already in registry.redhat.io/advanced-cluster-security/. Why? Simply, we must _release_ FBCs to customers with image references to the Red Hat Container Catalog which they can work with. That's why we pretend the image is there.

I suggested to do it as part of the same task https://issues.redhat.com/browse/ROX-26601 and I'm still with an opinion that it should be done (not dropped or postponed). The reason is that we'll simulate what we'll do at the time of the release and we'll be able to catch any unknown issues early and not at the time of the release.

This activity would not block this PR but I want to mention it anyway.

Reply:
I have removed the ICSP from this PR, but for completeness of the testing instructions I will also add the (updated) ICSP the this PR's description.